### PR TITLE
util: fix typo `s/goods/good`

### PR DESCRIPTION
**Pull request by bestgopher**
- I think this is a typo, but I ’m not sure.
- The error has been checked in the first few lines.

### Release note

- N/A

### CODE REVIEW

File: util/prefix_helper.go [hunk 1]
```diff
@@ -33,10 +33,6 @@ func ScanMetaWithPrefix(retriever kv.Retriever, prefix kv.Key, filter func(kv.Ke
 	defer iter.Close()
 
 	for {
-		if err != nil {
```
Comment (bestgopher) [inline util/prefix_helper.go:36]: The error has been checked on line 30.

File: util/set/int_set.go [hunk 2]
```diff
@@ -17,8 +17,12 @@ package set
 type IntSet map[int]struct{}
 
 // NewIntSet builds a IntSet.
-func NewIntSet() IntSet {
-	return make(map[int]struct{})
+func NewIntSet(ss ...int) IntSet {
+	set := make(IntSet, len(ss))
+	for _, x := range ss {
+		set.Insert(x)
+	}
```
Comment (zimulala) [inline util/set/int_set.go:24]: Why change this code?

Comment (bestgopher) [reply util/set/int_set.go:24]:
```
// NewStringSet builds a string set.
func NewStringSet(ss ...string) StringSet {
	set := make(StringSet, len(ss))
	for _, s := range ss {
		set.Insert(s)
	}
	return set
}

// NewFloat64Set builds a float64 set.
func NewFloat64Set(fs ...float64) Float64Set {
	x := make(map[float64]struct{}, len(fs))
	for _, f := range fs {
		x[f] = struct{}{}
	}
	return x
}

// NewInt64Set builds a Int64Set.
func NewInt64Set(xs ...int64) Int64Set {
	set := make(Int64Set, len(xs))
	for _, x := range xs {
		set.Insert(x)
	}
	return set
}
```

Comment (zimulala) [reply util/set/int_set.go:24]: Fine.

**Pull request closed without merging**